index.d.ts

```diff
@@ -0,0 +1,85 @@
+type SearchClient = object;
+
+interface TypesenseNode {
+  host: string;
+  port: string;
```

Comment:
Port should be always a number. You can convert it to string whenever you want and when it makes sense, or if you want to use it in connection string. But from the networking point of view, port is always a number and it should be handled like that.

Reply:
Yeah true actually but in the constructor it is expecting a `string`.

Reply:
Well then it's okay :)